tests/key_field/key_field_sample_generated.h

```diff
@@ -170,13 +200,362 @@ inline bool operator!=(const Bar &lhs, const Bar &rhs) {
 }
 
 
+FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(4) Color FLATBUFFERS_FINAL_CLASS {
+ private:
+  float rgb_[3];
+  uint8_t tag_;
+  int8_t padding0__;  int16_t padding1__;
+
+ public:
+  static const flatbuffers::TypeTable *MiniReflectTypeTable() {
+    return ColorTypeTable();
+  }
+  Color()
+      : rgb_(),
+        tag_(0),
+        padding0__(0),
+        padding1__(0) {
+    (void)padding0__;
+    (void)padding1__;
+  }
+  Color(uint8_t _tag)
+      : rgb_(),
+        tag_(flatbuffers::EndianScalar(_tag)),
+        padding0__(0),
+        padding1__(0) {
+    (void)padding0__;
+    (void)padding1__;
+  }
+  Color(flatbuffers::span<const float, 3> _rgb, uint8_t _tag)
+      : tag_(flatbuffers::EndianScalar(_tag)),
+        padding0__(0),
+        padding1__(0) {
+    flatbuffers::CastToArray(rgb_).CopyFromSpan(_rgb);
+    (void)padding0__;
+    (void)padding1__;
+  }
+  const flatbuffers::Array<float, 3> *rgb() const {
+    return &flatbuffers::CastToArray(rgb_);
+  }
+  flatbuffers::Array<float, 3> *mutable_rgb() {
+    return &flatbuffers::CastToArray(rgb_);
+  }
+  bool KeyCompareLessThan(const Color * const o) const {
+    return KeyCompareWithValue(o->rgb()) < 0;
+  }
+  int KeyCompareWithValue(const flatbuffers::Array<float, 3> *_rgb) const {
```

Comment:
is the `_rgb` for preventing name clashes? I think just `rgb` should work

Reply:
'rgb' clashes with 'rgb()' method, emit error